pkg/config/logs/log.go

```diff
@@ -108,10 +110,26 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 	}
 	_ = seelog.ReplaceLogger(loggerInterface)
 	log.SetupLogger(loggerInterface, seelogLogLevel)
-	scrubber.AddStrippedKeys(cfg.GetStringSlice("flare_stripped_keys"))
+	flareStrippedKeys := cfg.GetStringSlice("flare_stripped_keys")
+	if len(flareStrippedKeys) > 0 {
+		log.Warn("flare_stripped_keys is deprecated, please use scrubber.additional_keys instead.")
+	}
+	scrubber.AddStrippedKeys(mergeAdditionalKeysToScrubber(
+		flareStrippedKeys,
+		cfg.GetStringSlice("scrubber.additional_keys")))
 	return nil
 }
 
+// mergeAdditionalKeysToScrubber merges multiple slices of keys into a single slice
+func mergeAdditionalKeysToScrubber(elems ...[]string) []string {
+	set := []string{}
+	for _, elem := range elems {
+		clone := slices.Clone(elem)
```

Comment:
Since you're adding to a new slice, `set`, you  don't need to clone the current list.

Reply:
That what I thought